cmd/utils/utils.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package utils
+
+import (
+	"time"
+
+	"github.com/briandowns/spinner"
+)
+
+var DefaultSpinner = spinner.New(spinner.CharSets[9], 50*time.Millisecond)
```

Comment:
It is a lot slower in `az` 😅

Reply:
I agree I got bit aggressive with that  :see_no_evil: seems like they are using [200 ms frequency](https://github.com/xolox/python-humanfriendly/blob/master/humanfriendly/terminal/spinners.py#L110) so I will switch to the same!